packages/infra/lib/shared/rds.ts

```diff
@@ -53,32 +62,41 @@ export function addDBClusterPerformanceAlarms(
   createAlarm({
     metric: dbCluster.metricCPUUtilization(),
     name: "CPUUtilizationAlarm",
-    threshold: thresholds.cpuUtilizationPct,
+    threshold: dbConfig.alarmThresholds.cpuUtilizationPct,
     evaluationPeriods: 1,
     treatMissingData: cloudwatch.TreatMissingData.NOT_BREACHING,
   });
 
   createAlarm({
     metric: dbCluster.metricVolumeReadIOPs(),
     name: "VolumeReadIOPsAlarm",
-    threshold: thresholds.volumeReadIops,
+    threshold: dbConfig.alarmThresholds.volumeReadIops,
     evaluationPeriods: 1,
     treatMissingData: cloudwatch.TreatMissingData.NOT_BREACHING,
   });
 
   createAlarm({
     metric: dbCluster.metricVolumeWriteIOPs(),
     name: "VolumeWriteIOPsAlarm",
-    threshold: thresholds.volumeWriteIops,
+    threshold: dbConfig.alarmThresholds.volumeWriteIops,
     evaluationPeriods: 1,
     treatMissingData: cloudwatch.TreatMissingData.NOT_BREACHING,
   });
 
   createAlarm({
     metric: dbCluster.metricACUUtilization(),
     name: "ACUUtilizationAlarm",
-    threshold: thresholds.acuUtilizationPct,
+    threshold: dbConfig.alarmThresholds.acuUtilizationPct,
+    evaluationPeriods: 1,
+    treatMissingData: cloudwatch.TreatMissingData.NOT_BREACHING,
+  });
+
+  createAlarm({
+    metric: dbCluster.metricDatabaseConnections(),
+    name: "DatabaseConnectionsAlarm",
+    threshold: 0.8 * getMaxPostgresConnections(dbConfig.maxCapacity),
```

Comment:
Can we please have the 80% threshold as a const at the top of the file?

Reply:
Won't be constant because I need to use the fn args but I can lift to the top of the fn